components/scream/src/share/util/scream_kokkos_utils.hpp

```diff
@@ -2,12 +2,16 @@
 #define SCREAM_KOKKOS_UTILS_HPP
 
 #include <share/scream_kokkos_meta.hpp>
+#include <share/scream_types.hpp>
 #include <share/util/scream_std_meta.hpp>
 
 #include <Kokkos_Core.hpp>
 
 #include <cassert>
 #include <type_traits>
+#ifdef _OPENMP
```

Comment:
Also remove this `#include <omp.h>` block when you switch _OPENMP to KOKKOS_ENABLE_OPENMP below. Thanks.